### PR TITLE
Implement SAIC-36 : snapshots support of CF

### PR DESCRIPTION
**Pull request by oleksii-shyman**
- added task that makes dumps of mysql and copies them to remote host
  - added task that applies mysqldump to existing database
  - extended config.ini with [snapshot] section where settings are
    stored

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/86/

File: cloudferrylib/os/actions/snapshots.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+from cloudferrylib.utils import utils
+
+
+LOG = utils.get_log(__name__)
+
+
+class MysqlDump(action.Action):
+
+    def __init__(self, init, cloud):
+        super(MysqlDump, self).__init__(init, cloud)
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/snapshots.py]: Looks like constructor is not really needed here, you're only calling the base class.

Comment (oleksii-shyman) [inline cloudferrylib/os/actions/snapshots.py]: true

File: cloudferrylib/os/actions/snapshots.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        ssh_file_to_file.SSHFileToFile(self.src_cloud, self.dst_cloud,
+                                       self.cloud.cloud_config).transfer(
+            context)
+        return {}
+
+
+class MysqlRestore(action.Action):
+
+    def __init__(self, init, cloud):
+        super(MysqlRestore, self).__init__(init, cloud)
```
Comment (roman-verchikov) [inline cloudferrylib/os/actions/snapshots.py]: Same as above

**Comment by roman-verchikov** [comment]
Looks good other than few minor comments inline

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/87/

**Pull request merged**